src/toast/Toast.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import "@clr/icons/clr-icons.min.css";
+import "@webcomponents/custom-elements/custom-elements.min.js";
+import "@clr/icons/clr-icons.min.js";
+import {Icon} from "../icon";
+import "./styles.css";
+
+export default class Toast extends React.Component<Props> {
+    constructor(props: Props) {
+        super(props);
+        props.notify(this.launch_toaster);
+        this.state = {
+            showToast: false,
+            msg: "",
+            type: "success",
+            position: "top-right",
+            hideAfter: 3,
+            style: {opacity: 0},
+        };
+    }
+    /* Show the toaster unless another toaster is already in display */
+    launch_toaster = (message: string, options: object) => {
+        const hideTime = (options.hideAfter === undefined ? 3 : options.hideAfter) * 1000;
+        const toastType = options.type === undefined ? "success" : options.type;
+        const pos = options.position === undefined ? "top-right" : options.position;
+
+        if (this.state.showToast === false) {
```

Comment:
Should this be vice-versa, I mean default should be true, and when we need to wait till it fades, it should be false.